LayoutTests/mathml/presentation/foreign-element-in-token-expected.txt

```diff
@@ -455,18 +455,73 @@ mo
 Foreign non-phrasing content should not create renderers:
```

Comment:
Can we update the text in foreign-element-in-token.html now that foreign non-phrasing content do create renderers?